src/Components/Header.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+interface HeaderPropsType {
+  todos: Todo[],
+  searchQuery: string,
+  setSearchQuery: (searchQuery: string) => void,
+}
+
+export const Header: React.FC<HeaderPropsType> = ({
+  todos,
+  searchQuery,
+  setSearchQuery,
+}) => {
+  const isActicve = todos.filter(todo => !todo.completed);
```

Comment:
Consider renaming `isActicve` to `isActive` to correct the typo and improve code readability 🚀